ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -89,8 +103,5 @@ export function createInitialTimeSeriesChartOptions(): TimeSeriesChartOptions {
         },
       },
     ],
-    unit: {
```

Comment:
`unit` moves inside `y_axis`, I removed it from initial options here since all the new properties are optional (most people can rely on defaults and less JSON for GitOps)

Reply:
Oh yeah that sounds nice